### PR TITLE
Add motion.reset_state action + tests.

### PR DESCRIPTION
**Pull request by reiterl**

### CODE REVIEW

File: openslides_backend/action/motion/reset_state.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    model = Motion()
+
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        """
+        Set state_id to motion_state.first_state_of_workflow_id.
+        """
+        motion = self.database.get(
+            FullQualifiedId(Collection("motion"), instance["id"]), ["state_id"]
+        )
+        if motion.get("state_id"):
```
Comment (jsangmeister) [inline openslides_backend/action/motion/reset_state.py:42]: Throw an error in the else part, a motion must always have a state.

**Review by jsangmeister — CHANGES_REQUESTED** [review]

File: openslides_backend/action/motion/reset_state.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
+from typing import Any, Dict
+
+from ...models.models import Motion
+from ...shared.exceptions import ActionException
+from ...shared.patterns import Collection, FullQualifiedId
+from ...shared.schema import schema_version
+from ..generics import UpdateAction
+from ..register import register_action
+
+reset_schema = {
```
Comment (jsangmeister) [inline openslides_backend/action/motion/reset_state.py:10]: Please just use the default update schema without any properties.

**Review by jsangmeister — CHANGES_REQUESTED** [review]

File: openslides_backend/action/motion/reset_state.py (hunk 1 above)
Comment (reiterl) [reply openslides_backend/action/motion/reset_state.py:27]: Done.

File: openslides_backend/action/motion/reset_state.py (hunk 2 above)
Comment (reiterl) [reply openslides_backend/action/motion/reset_state.py]: Done. Use default update schema.

**Review by jsangmeister — APPROVED** [review]

**Comment by jsangmeister** [comment]
rebase

**Pull request merged**